### ISSUE TITLE
Possible small fix for example ipynb

### ISSUE THREAD

**Issue by rhamnett** [jdb78/pytorch-forecasting#53]
In https://github.com/jdb78/pytorch-forecasting/blob/master/docs/source/tutorials/stallion.ipynb cell 9, should learning rate not be set to the result of the study in cell 8 or am I missing something?

```
# configure network and trainer
early_stop_callback = EarlyStopping(monitor="val_loss", min_delta=1e-4, patience=10, verbose=False, mode="min")
lr_logger = LearningRateLogger()  # log the learning rate
logger = TensorBoardLogger("lightning_logs")  # logging results to a tensorboard

trainer = pl.Trainer(
    max_epochs=30,
    gpus=0,
    weights_summary="top",
    gradient_clip_val=0.1,
    early_stop_callback=early_stop_callback,
    limit_train_batches=30,  # coment in for training, running valiation every 30 batches
    # fast_dev_run=True,  # comment in to check that networkor dataset has no serious bugs
    callbacks=[lr_logger],
    logger=logger,
)


tft = TemporalFusionTransformer.from_dataset(
    training,
    learning_rate=res.suggestion(), #<<<<<<<<<<<<<<<<<<<<<<<<<<<< this
    hidden_size=16,
    attention_head_size=1,
    dropout=0.1,
    hidden_continuous_size=8,
    output_size=7,  # 7 quantiles by default
    loss=QuantileLoss(),
    log_interval=30,  # uncomment for learning rate finder and otherwise, e.g. to 10 for logging every 10 batches
    reduce_on_plateau_patience=4,
)
print(f"Number of parameters in network: {tft.size()/1e3:.1f}k")
```

**Comment by jdb78** [comment]
Thanks for your comments! I believe it makes sense to be clearer in the tutorial.

Generally, I have found that setting a slightly smaller learning rate than suggested works better. Using the learning rate finder and using its result automatically can be also very dangerous because sometimes the suggestion is in the noisy region or the graph. Controlling it manually is the safer way.

**Comment by jdb78** [comment]
Added explanation in PR #54
